### ISSUE TITLE
Version 3.15.0 for linux-amd64 is not published.

### ISSUE THREAD

**Issue by m407** [helm/helm#13060]
Version 3.15.0 for linux-amd64 is not published.
**_helm-v3.15.0-linux-amd64.tar.gz.asc_** exists but **_helm-v3.15.0-linux-amd64.tar.gz_** not

**Comment by den-is** [comment]
well archive exists.. it is in a release description
But thing it is that packaged binary reports wrong version
```sh
curl -OL https://get.helm.sh/helm-v3.15.0-linux-amd64.tar.gz
tar xzf ...; cd linux-amd64
./helm version
version.BuildInfo{Version:"v3.15.0-rc.2", GitCommit:"c4e37b39dbb341cb3f716220df9f9d306d123a58", GitTreeState:"clean", GoVersion:"go1.22.3"}
```

**Comment by mattfarina** [comment]
There was a bug in the build process where it injects the version. Both the RC and the final release point to the same commit. For the first time we are aware of, the detection of the version picked up the wrong one. We fixed this and just released 3.15.1.
